remote/bitbucketserver/internal/client.go

```diff
@@ -12,18 +12,20 @@ import (
 	log "github.com/Sirupsen/logrus"
 	"github.com/drone/drone/model"
 	"github.com/mrjones/oauth"
+	"strings"
```

Comment:
the `strings` package should be grouped with the other standard library packages. This is a recommended convention for Go projects.

```diff
import (
	"bytes"
	"encoding/json"
	"fmt"
	"io"
	"io/ioutil"
	"net/http"
	"strconv"
+	"strings"

	log "github.com/Sirupsen/logrus"
	"github.com/drone/drone/model"
	"github.com/mrjones/oauth"
-	"strings"
)
```